arches_for_science/media/js/views/components/workflows/create-project-workflow/add-things-step.js

```diff
@@ -273,8 +334,11 @@ define([
                     self.termOptions = results;
 
                     const filteredResults = results.filter(function(result){
-                        return result.context_label.includes("Physical Thing") ||
-                        result.context_label.includes("Search Term");
+                        return (
+                            result.context_label.includes("Physical Thing") ||
+                            result.context_label.includes("Search Term")
+                        ) && (self.includeSamples() || !result.text.includes(sampleSubstring))
+                        && (self.includeAnalysisAreas() || !result.text.includes(regionSubstring))
```

Comment:
I still found substring matching necessary to filter out children from the select2 widget, as I don't have access to concepts/tiles at that point. Open to other ideas.

Reply:
Maybe we could do something like this on the Arches side: https://github.com/archesproject/arches/pull/10216/files

Reply:
I don't think the graphid will help, because both samples and their parents will share the same graph id for physical thing. AFAICT I would need to drill in to the tile data along the lines of what I did in the middle of the long sort function added in this PR.

Reply:
I see - right, it's the distinction between types of physical things that is needed, not resource models.